core/controllers/creator_dashboard_test.py

```diff
@@ -501,10 +501,10 @@ def test_get_suggestions_to_review_list(self) -> None:
             feconf.CREATOR_DASHBOARD_DATA_URL)['suggestions_to_review_list']
 
         self.assertEqual(len(suggestions), 1)
-        self.assertEqual(suggestions[0]['change'], change_dict)
+        self.assertEqual(suggestions[0]['change_cmd'], change_dict)
         # Test to check if suggestions populate old value of the change.
         self.assertEqual(
-            suggestions[0]['change']['old_value']['content_id'], 'content_0')
+            suggestions[0]['change_cmd']['old_value']['content_id'], 'content_0') # pylint: disable=line-too-long
```

Comment:
You should avoid using pylint disable where possible. You can maybe shift `content_0` to next line.

Reply:
Done